README.md

```diff
@@ -5,7 +5,10 @@ A tool for enriching the output of `nvidia-smi`.
 ![CI](https://github.com/peci1/nvidia-htop/workflows/CI/badge.svg)
 
 # Install
-`pip3 install .`
+
+```bash
+pip install git+https://github.com/peci1/nvidia-htop.git
```

Comment:
pip3 ?

Reply:
Yes, pip3. Python2.x will not be maintained, so by default, I think normally we use pip3 to install libraries.